### PR TITLE
Clean up unused dependencies & add a sample project

### PR DESCRIPTION
**Pull request by skydoves**
### 🎯 Goal

Clean up unused dependencies & add a sample project.

### CODE REVIEW

File: config/detekt/detekt-compose.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+naming:
```
Comment (JcMinarro) [inline config/detekt/detekt-compose.yml:1]: Are we using this `detekt-compose` file anywhere? :thinking:

File: config/detekt/detekt-test.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+complexity:
```
Comment (JcMinarro) [inline config/detekt/detekt-test.yml:1]: I think this `detekt-test` config is not used into this project

**Review by JcMinarro — APPROVED** [review]
Looks good.
I added some comments about the extra Detekt config that I think we are not using

**Comment by skydoves** [comment]
Thanks for the quick comments, @JcMinarro!
I removed unused Detekt files: [b3d472a](https://github.com/GetStream/stream-log/pull/2/commits/b3d472a595abdb224681a166bc962fe38bdbec22)

**Pull request merged**